### PR TITLE
add post about new privacy policy

### PR DESCRIPTION
**Pull request by fmichonneau**

### CODE REVIEW

File: _posts/2022/01/2022-01-15-updated-privacy-policy.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+teaser: "We are updating Privacy Policy and Terms & Conditions"
+title: "Update to our Privacy Policy"
+date: 2022-01-17
+time: "09:00:00"
+tags: ["Governance"]
+---
+
+
+We are pleased to announce the release our new [Privacy
+Policy](https://docs.carpentries.org/topic_folders/policies/privacy.html). This
```
Comment (kmomar) [inline _posts/2022/01/2022-01-15-updated-privacy-policy.md:13]: Will this be updated before I publish the blog? Currently says May 2018

File: _posts/2022/01/2022-01-15-updated-privacy-policy.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+new version follows a more standard structure and spells out your rights and how
+to exercise them under European Union's General Data Protection Regulation
+(GDPR) and the California Customer Privacy Act (CCPA). It also includes the
```
Comment (kmomar) [inline _posts/2022/01/2022-01-15-updated-privacy-policy.md:16]: Can we link to GDPR and CCPA sites?

File: _posts/2022/01/2022-01-15-updated-privacy-policy.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+provide you with access to all the data we store about you directly from your
+AMY profile.
+
+As part of reviewing our practices, we have switched the analytics of our
+websites to use [Matomo](https://matomo.org). With Matomo, the analytics data
+belong to us.
+
+Cookie consent banners hamper the user experience and we therefore decided to
+only use strictly necessary cookies across all our sites, even if it means that
+some of our analytics is not as accurate or detailed as it could be.
```
Comment (kmomar) [inline _posts/2022/01/2022-01-15-updated-privacy-policy.md]: some of our analytics are not as accurate or detailed as they could be.

**Review by kmomar — CHANGES_REQUESTED** [review]

File: _posts/2022/01/2022-01-15-updated-privacy-policy.md (hunk 1 above)
Comment (fmichonneau) [reply _posts/2022/01/2022-01-15-updated-privacy-policy.md:13]: yes, I'm going to merge the PRs with the new docs now, so the docs will be published when the post goes out.

File: _posts/2022/01/2022-01-15-updated-privacy-policy.md (hunk 2 above)
Comment (fmichonneau) [reply _posts/2022/01/2022-01-15-updated-privacy-policy.md:16]: there are not really sites we can link to for these are they are laws. They are sites with the law but they are commercial.

File: _posts/2022/01/2022-01-15-updated-privacy-policy.md (hunk 3 above)
Comment (fmichonneau) [inline _posts/2022/01/2022-01-15-updated-privacy-policy.md]:
```suggestion
some of our analytics are not as accurate or detailed as it could be.
```

**Pull request merged**